cdp/address.py

```diff
@@ -98,6 +101,33 @@ def balances(self):
 
         return BalanceMap.from_models(response.data)
 
+    def historical_balances(self, asset_id) -> Iterator["HistoricalBalance"]:
```

Comment:
```suggestion
    def historical_balances(self, asset_id) -> Iterator[HistoricalBalance]:
```

These should only need to be in quotes if they are in the file where the class is defined.